packages/web/src/pages/app/repositories.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { NextPage } from 'next';
+import {
+  Input,
+  Button,
+  Link,
+  Heading,
+  useTheme,
+  VStack,
+  ListItem,
+  Box,
+  SimpleGrid,
+  FormControl,
+} from '@chakra-ui/react';
+import { AppLayout } from '../../components/Layout';
+import { RepoBox } from '../../components/Repos';
+
+export interface RepoList {
+  id: React.Key | undefined;
```

Comment:
`React.Key` might be the type for the `key` property, but in this case it should represent what is coming back from the server, which would be a string

```suggestion
  id: string;
```